ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala

```diff
@@ -413,12 +413,10 @@ final class SandboxServer(
         )
       }
       if (config.seeding.isEmpty) {
-        // TODO https://github.com/digital-asset/daml/issues/7139
-        //  rephrase the message once LF 1.11 is released
         logger.withoutContext.warn(
-          s"""|'${Seeding.NoSeedingModeName}' contract IDs seeding mode is not compatible with the LF 1.11 languages or later which will be released soon.
+          s"""|'${Seeding.NoSeedingModeName}' contract IDs seeding mode is not compatible with the LF 1.11 languages or later.
```

Comment:
My understanding was that @bame-da and you agreed to not have a default once 1.11 is stable and force users to choose. Is that still the plan? Perfectly fine not to include it in this PR.

Reply:
I did not understand that. Let's sync with @bame-da and implement a fix (if necessary) in another PR.